ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -58,6 +67,16 @@ DD:
   - heating_air_flowrate_setpoint
   implements:
   - CONTROL
+  
+MIOVCM:
+  description: "Motor input-output current and voltage monitoring."
+  is_abstract: true
+  implements:
+  - MONITORING
+  uses:
+  - dc_voltage_sensor
+  - ac_voltage_sensor
+  - current_sensor  
```

Comment:
Which side? If you are differentiating between ac and dc sides, you must differentiate here and include both.